src/features/Kahoot/index.tsx

```diff
@@ -1,11 +1,22 @@
+import { AppDispatch } from '@/app/store';
 import NotFound from '@/components/NotFound';
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
 import { Route, Switch, useRouteMatch } from 'react-router-dom';
 import Creator from './Creator';
+import { getKahoots } from './kahootSlice';
 import MyKahoot from './MyKahoot';
 
 function Kahoot() {
   const match = useRouteMatch();
+  const dispatch = useDispatch<AppDispatch>();
+
+  useEffect(() => {
+    async function fetchKahootList() {
```

Comment:
Unnecessary to create a function fetchKahootList. Just use dispatch(getKahoots()) directly ;